tests/ctests/CMakeLists.txt

```diff
@@ -5,15 +5,13 @@ file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/ctests/*.cpp")
 
 # setup test executable
 add_executable(runAleTests ${test_source})
-target_link_libraries(runAleTests ale ${GSL_LIBRARIES} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} pthread)
-
-target_include_directories(runAleTests
-                           PRIVATE
-                           ${GSL_INCLUDE_DIRS}
-                           PUBLIC
-                           ${GSL_INCLUDE_DIRS}
-                           ${EIGEN3_INCLUDE_DIR}
-                           ${ALE_INCLUDE_DIRS})
-
+target_link_libraries(runAleTests
+                       ale
+                       GSL::gsl
+                       GSL::gslcblas
+                       Eigen3::Eigen
+                       ${GTEST_LIBRARIES}
+                       ${GTEST_MAIN_LIBRARIES}
+                       pthread)
```

Comment:
Any particular reason for making this explicit? Or is this change to fix something else?

Reply:
The ale header is simply dropped into the root of the include dirs in the install location, meaning anytime an API change was made, you had to manually delete ale.hpp in the install directory as it was always picked up first as the include resolution order has third-party includes that also exist in the same location, preventing your local ale.hpp from being included. This should always have Ale's local includes first.